### ISSUE TITLE
[fill-ratio] examples/test scripts need updates (Trac #1200)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1194]
- FillRatioExample.py should be moved to ../resources/examples
- FillRatioTest.py should be moved to ../resources/tests
- python/test_modules/FRTestModule.py should be moved to ../resources/tests or be included into FillRatioTest.py
- All test and examples scripts need updates; they still use the old tray python syntax
- More tests are needed

Migrated from https://code.icecube.wisc.edu/ticket/1200
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "- FillRatioExample.py should be moved to ../resources/examples\n- FillRatioTest.py should be moved to ../resources/tests\n- python/test_modules/FRTestModule.py should be moved to ../resources/tests or be included into FillRatioTest.py\n- All test and examples scripts need updates; they still use the old tray python syntax\n- More tests are needed", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[fill-ratio] examples/test scripts need updates", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T18:33:48", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T12:35:35: 
* mjl5147 changed owner from "mlarson" to "mjl5147"
* mjl5147 changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160224T19:47:44: 
* mjl5147 changed status from "assigned" to "closed"
* mjl5147 changed resolution from "" to "fixed"
* mjl5147 commented:

> In changeset 142445:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142445"
> Updated the test script to finally test the I3FillRatioLite.cxx. This should be the last big test update for today and this closes #1194.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
